### ISSUE TITLE
Demo not working in IE 11

### ISSUE THREAD

**Issue by Compufreak345** [Polymer/polymer#3180]
It seems like a recent update broke Polymer for IE11 - my own project does not load anymore and also the demos seem to have issues. 

One example : 
https://elements.polymer-project.org/elements/paper-badge?view=demo:demo/index.html&active=paper-badge
![ie11_broke](https://cloud.githubusercontent.com/assets/10026694/11685721/5ea68df8-9e7b-11e5-8a1d-d296b88175e1.png)

**Comment by lismore** [comment]
I have been having issues with IE this last two days,  I found a solution that works for me,  maybe the feedback can help in a fix. 

I have a dom-module custom element that loads in a partial view (.NET MVC View) through jquery.  However IE 11 and < 11 it would not load at all, no matter what I tried.  

In the end I found that I had to wrap the jquery call in a setTimeout for it to wait until polymer custom element had loaded. Even though on screen it was visually there.  I found I had to do this in a few custom elements so it would work in IE.

Not sure if this is a polymer issue or IE.  At least now I know what to look for.

setTimeout(function() {

                // load the partial section
                jQuery(".partial-view").each(function (index, item) {
                    var url = jQuery(item).data("url");
                    if (url && url.length > 0) {
                        jQuery(item).load(url);
                    }
                });

            }, 2000);
